packages/terra-action-footer/docs/CenteredActionFooter.md

```diff
@@ -0,0 +1,39 @@
+# Terra Centered Action Footer
+
+The terra-centered-action-footer component is a footer bar that contains a single, centered socket for placing actionable items such as buttons and hyperlinks. If no actions are provided, the footer bar collapses to a themeable height (default of 10px) and maintains the top border.
```

Comment:
```
(default of 10px)
```

I would avoid calling out specific values in the read me documentation because it often times gets out of sync. I think the default value has already been converted into rems which will convert to a different px value depending on the themed font-size